Source/Core/DolphinQt2/Settings.cpp

```diff
@@ -150,3 +150,31 @@ QVector<QString> Settings::GetProfiles(const InputConfig* config) const
 
   return vec;
 }
+
+bool Settings::IsLogVisible() const
+{
+  return SConfig::GetInstance().m_InterfaceLogWindow;
```

Comment:
Yeah, you can follow what #5755 is doing and create a temporary `QSettings()` (they're all hooked together internally).